src/procedures/procedures.11tydata.yaml

```
@@ -367,6 +367,152 @@ proceduresInstallingHAOS:
+      content: |
+        <ol>
+          <li>On the Yellow board, press and <b>hold</b> the recovery button.
+            <ul><li>Older Home Assistant Yellow versions: Bridge JP2 (USB-C Rcvry).</li></ul>
+          </li>
+          <li>Power up the board.
+          <ul><li>Only the red LED should be on at this point (solid, not blinking).</li></ul>
+          </li>
+          <li>Release the recovery push button.
+            <ul><li>Older Home Assistant Yellow versions: Un-bridge JP2.</li>
+            </ul>
+          </li>
+        </ol>
+    - title: Run rpiboot
+      image: led-pattern-install-green.png
+      content: |
+        <ul>
+          <li>On your PC, run rpiboot and let it run through.
+            <ul>
+              <li>If you are on a Windows, it might ask you if you want to reformat the disk. Select <b>Cancel</b> each time.</li>
+              <li>If you are on a linux, use <code>sudo ./rpiboot -d mass-storage-gadget64</code> to run the tool.</li>                           
+              <li>If you are running an Apple Silicon mac,  before running rpiboot, make sure you open the terminal using the “Rosetta” compatibility mode.
+                </li>              
+            </ul>
+          </li>
+          <li>After a few seconds, the yellow LED on the Yellow board should start blinking.</li>
+          <li>Afterwards, only the green LED should be on.</li>
+          <li>The module is now ready and to have the Home Assistant Operating System installed on it (next steps).</li>      
+        </ul>
+proceduresRunningRpiImagerCM5:
+  title: Installing Home Assistant Software on CM5
+  steps:
+    - title: Installing the Raspberry Pi Imager
+      image: rpi_imager.png
+      content: |
+        <ul>
+          <li>Install the Raspberry Pi Imager on your computer as described under <a href="https://www.raspberrypi.com/software/" target="_blank">https://www.raspberrypi.com/software/</a></li>
+          <li class="info">If your platform doesn't support the Raspberry Pi Imager, you can use another imager (e.g. balenaEtcher) and flash the Home Assistant OS Installer for Yellow directly from <a href="https://github.com/NabuCasa/yellow-buildroot/releases">GitHub releases</a>.</li>
+        </ul>
+    - title: Open Raspberry Pi Imager      
+      image: yellow_rpi-imager_select_device_cm5.png
+      content: |
+        <ul>
+          <li>Open Raspberry Pi Imager.</li>
+          <li>Under <b>Raspberry Pi Device</b>, select </b>Raspberry Pi 5</b>.</li>     
+        </ul>
+    - title: Choosing the operating system
+      image: yellow_rpi-imager_cm5_module_image.webp
+      content: |
+        <ul>
+          <li>Select <b>Choose OS</b></li>
+          <li>Choose <b>Other specific-purpose OS</b>.</li>
+          <li>Choose <b>Home assistants and home automation</b>.</li>
+          <li>Choose <b>Home Assistant</b>.</li>
+          <li>Choose <b>Home Assistant OS 14.1 (Yellow)</b>.</li>
```

This will quickly become outdated, so I'd leave it open e.g. like this?
```suggestion
          <li>Choose <b>Home Assistant OS x.y (Yellow)</b>.</li>
```